src/core/variable.jl

```diff
@@ -762,6 +762,40 @@ function variable_mc_switch_state(pm::AbstractUnbalancedPowerModel; nw::Int=nw_i
     report && _IM.sol_component_value(pm, pmd_it_sym, nw, :switch, :state, ids(pm, nw, :switch_dispatchable), state)
 end
 
+"capacitor (with capcontrol) switch state (open/close) variables"
+function variable_mc_capacitor_switch_state(pm::AbstractUnbalancedPowerModel; nw::Int=nw_id_default, report::Bool=true, relax::Bool=false)
+    cap_switch_ids = [id for (id,cap) in ref(pm, nw, :shunt) if haskey(cap,"controls")]
+    if relax
+        cap_state = var(pm, nw)[:capacitor_state] = Dict(i => JuMP.@variable(pm.model,
+        [p in ref(pm, nw, :shunt, i, "connections")],
+        base_name="$(nw)_cap_sw_$(i)",
+        lower_bound = 0,
+        upper_bound = 1,
+        start = 0.5
+        ) for i in cap_switch_ids)
+    else
+        cap_state = var(pm, nw)[:capacitor_state] = Dict(i => JuMP.@variable(pm.model,
+        [p in ref(pm, nw, :shunt, i, "connections")],
+        base_name="$(nw)_cap_sw_$(i)",
+        binary = true,
+        start = 1
+        ) for i in cap_switch_ids)
+    end
+
+    report && _IM.sol_component_value(pm, pmd_it_sym, nw, :shunt, :cap_state, cap_switch_ids, cap_state)
+end
+
+
+"capacitor (with capcontrol) relaxed current variables for linear models"
```

Comment:
should correct to follow new conventions for docstring format that include the function signature